lib/tapi/tapi_cfg_memory.c

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/** @file
+ * @brief Test API to configure RAM.
+ *
+ * Implementation of API to configure RAM.
+ *
+ * Copyright (C) OKTET Labs Ltd. All rights reserved.
```

Comment:
- Copyright should be a non-doxygen comment right after SPDX identifier
-  I believe, the copyright is of OKTET or ARK Networks, not OKTET Labs

Reply:
Copyright should be OKTET.